libs/utils/analysis/latency_analysis.py

```diff
@@ -49,6 +51,26 @@ def __init__(self, trace):
 
     @memoized
     def _dfg_latency_df(self, task):
+        """
+        DataFrame of task's wakeup/suspend events
+
+        The returned DataFrame has these columns
+        - Time: the time an event related to this task happened
```

Comment:
As per f2f discussion: I think it should say something like "The index is the time, in seconds, an event related to this task happened". The index is not a column and I think the "average" user of pandas is required to understand this (and might be confused by it being listed like this).